lib/Logger.js

```diff
@@ -14,6 +14,8 @@ module.exports = class Logger {
     this._transports = transports;
     this._threshold = level;
     this._disabled = false;
+    this._promises = new Set();
+    this._draining = null;
```

Comment:
If decided to initialise the `_draining` property to make it obvious what properties this class supports